### ISSUE TITLE
Remove extraneous blank lines

### ISSUE THREAD

**Issue by pozix604** [pacak/cargo-show-asm#220]
`cargo asm` shows a lot of blank lines in my output. For example:

```
.LBB68_22:
	pop rbx
	pop r12
	pop r13
	pop r14
	pop r15
	ret

.LBB68_9:
	lea rdx, [rip + .L__unnamed_27]

	mov esi, 5248

	mov rdi, r8

	call qword ptr [rip + core::panicking::panic_bounds_check@GOTPCREL]

	ud2
```

I'd like to suggest that when `--simplify` is used to "Try to strip some of the non-assembly instruction information", as the help says, that it will also strip away these blank lines so that the output is more compact.

As a workaround, I currently do the following instead to get rid of blank lines. Folding this into `cargo asm` itself can benefit:

```
cargo asm --color --simplify --lib <function> | awk NF
```

**Comment by pacak** [comment]
This is partially intentional. `--simplify` strips unused labels, but leaves blank lines where single or multiple labels used to be. I find it that it separates them into somewhat logical blocks. Like in the example you posted a bunch of pop is a single block of instructions, llvm is not going to generate jumps inside. I can add a separate option I guess... Need to think about it a bit.

**Comment by jhpratt** [comment]
I was coming to suggest the same thing. My intention is comparing the number of instructions between two programs, which is more difficult when blank lines are included.

**Comment by pacak** [comment]
> My intention is comparing the number of instructions between two programs, which is more difficult when blank lines are included.

What's the motivation? In my experience number of instruction is rarely useful by itself. To minimize the size you should be looking at encoded size, to maximize the performance `llvm-mca` analysis is more useful (`cargo asm --mca-intel xxx`).

**Comment by pacak** [comment]
Released 0.2.23

**Comment by jhpratt** [comment]
Ooh, I wasn't aware of the `--mca-intel` flag. I am definitely using that. Thanks for the info! And the update, of course.
